hashdist/cli/frontend_cli.py

```diff
@@ -123,7 +123,16 @@ class Develop(ProfileFrontendBase):
     """
     Builds a development profile in the HashDist YAML profile spec
     format, at the same location as the profile yaml file, but without
-    the .yaml suffix.
+    the .yaml suffix. You can use the 'target' argument to name it differently
+    than your profile yaml file.
+
+    Example:
+
+        hit develop default.yaml xx
+
+    This creates a throw away developmet profile 'xx', into which you can
```

Comment:
> @cekees and I talked about actually storing the development profile inside the ~/.hashdist/bld directory as well (or perhaps under a separate directory, but inside ~/.hashdist

This is fine.  In this case, it is important to warn users that ~/.hashdist is a terrible place to put builds if they're on a parallel file system, since home directories tend to be not very performant.  

Another aspect of development profiles is that they can be actual copies, as opposed to just links.  This isn't a well-used or tested feature.